libcontainer/nsenter/namespace.h

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2014, 2015 Docker Inc.
+ * Copyright 2015, 2016 The Linux Foundation
```

Comment:
I don't think this is correct, the original copyright in the license stays as it is, it doesn't just stop because the repo changed.